composer.json

```diff
@@ -407,7 +407,7 @@
             "symfony local:server:start --dir=tests/e2e/update-api-mock --port=8060 -d"
         ],
         "ecs": "ecs check src tests",
-        "ecs-fix": "ecs check --fix src tests",
+        "ecs-fix": "ecs check --fix",
```

Comment:
why did you remove that? :thinking:

Reply:
oh sorry, my bad. I have removed this changes